addons/website_sale/tests/test_website_sale_shop_variant_exclusion.py

```diff
@@ -0,0 +1,92 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.base.tests.common import HttpCaseWithUserDemo, HttpCaseWithUserPortal
+from odoo.tests import tagged
+
+@tagged('post_install', '-at_install')
+class TestUi(HttpCaseWithUserDemo, HttpCaseWithUserPortal):
+
+    def setUp(self):
```

Comment:
FYI most of the time it's better to use `setUpClass` when writing tests so that it's only ran once per "file".
In this case it does not matter since there is only one test but it's a good thing to know.